app/src/androidTest/java/org/wikipedia/tests/SearchEngineTests.kt

```diff
@@ -72,6 +73,7 @@ class SearchEngineTests {
         articlePage.tapOnSearchBar()
         searchBarPage.tapOnDeleteRecentSearchesButton()
         searchBarPage.tapOnYesButton()
-        assertTrue(searchBarPage.isDisplayedEmptyPage())
+        //assertEquals(searchBarPage.getTextOfElement(), "Yes")
```

Comment:
nie wiem czemu nie dziala 😣

Reply:
to wiec tutaj byc potrzebowala pomocy RecycleView:
asercja na konkretna pozycje (0) ze ma text = Donald Trump

Reply:
ale zeby uzyc w assertEquals to co wyrzuci getText() z konkretnej pozycji